### PR TITLE
Improve cliinstall

### PR DESCRIPTION
**Pull request by remicollet**
1/ prompt for password to avoid having it saved in history/log
For security extremists, like @kapouik ;)

This change mimics the `mysql` command behavior.

2/ as zend-console is added for previous, use a modern and robust Getopt instead of the old trivial implementation

### CODE REVIEW

**Comment by remicollet** [comment]
Don't understand why travis fails...

So please only consider 1st commit....

**Comment by remicollet** [comment]
Ugly.. we cannot use Getopt "before" GLPI include (requested for autoload) to define so GLPI constant... :(

**Comment by remicollet** [comment]
Perhaps:
* 1st in 9.3
* others only in master...

File: scripts/cliinstall.php [hunk 1]
```diff
@@ -78,6 +89,10 @@ function displayUsage() {
 $_SESSION = ['glpilanguage' => (isset($args['lang']) ? $args['lang'] : 'en_GB')];
 Toolbox::setDebugMode(Session::DEBUG_MODE, 0, 0, 1);
 
+if ($args['pass'] === true) {
```
[the remark this answers is not in the document]
Comment (remicollet) [reply scripts/cliinstall.php]: when the option is given on the command line without parameter, it is considered as a Boolean option, ans thus, set to true.

[the remark this answers is not in the document]
Comment (trasher) [reply scripts/cliinstall.php]: oh, OK, thanks for the clarification :)

**Comment by trasher** [comment]
Just for you to know; I had in mind to create a kind of superclass for methods, with a common set of arguments; and I had in mind to use zend component for that. so this is indeed a first step ;)

**Pull request merged**